source/update.rst

```diff
@@ -32,33 +32,38 @@ Semi-automatic upgrade
 
    .. code:: bash
 
-      kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{{release}}}/deploy/crd.yaml
-      kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{{release}}}/deploy/rbac.yaml
+      $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{{release}}}/deploy/crd.yaml
+      $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v{{{release}}}/deploy/rbac.yaml
 
 #. Edit the ``deploy/cr.yaml`` file, setting ``updateStrategy`` key to
-   ``RollingUpdate``.
+   ``RollingUpdate``, and apply changes with the
+   ``kubectl apply -f deploy/cr.yaml`` command.
 
 #. Now you should `apply a patch <https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/>`_ to your
    deployment, supplying necessary image names with a newer version tag. This
    is done with the ``kubectl patch deployment`` command. For example, updating
-   to the ``{{{release}}}`` version should look as follows::
+   to the ``{{{release}}}`` version should look as follows:
 
-     kubectl patch deployment percona-server-mongodb-operator \
-        -p'{"spec":{"template":{"spec":{"containers":[{"name":"percona-server-mongodb-operator","image":"percona/percona-server-mongodb-operator:{{{release}}}"}]}}}}'
+   .. code:: bash
+
+      $ kubectl patch deployment percona-server-mongodb-operator \
+         -p'{"spec":{"template":{"spec":{"containers":[{"name":"percona-server-mongodb-operator","image":"percona/percona-server-mongodb-operator:{{{release}}}"}]}}}}'
 
-     kubectl patch psmdb my-cluster-name --type=merge --patch '{
-        "spec": {
+      $ kubectl patch psmdb my-cluster-name --type=merge --patch '{
+         "spec": {
             "crVersion":"{{{release}}}",
             "image": "percona/percona-server-mongodb:{{{mongodb44recommended}}}",
             "backup": { "image": "percona/percona-server-mongodb-operator:{{{release}}}-backup" },
             "pmm": { "image": "percona/pmm-client:{{{pmm2recommended}}}" }
-        }}'
+         }}'
 
 #. The deployment rollout will be automatically triggered by the applied patch.
    You can track the rollout process in real time using the
-   ``kubectl rollout status`` command with the name of your cluster::
+   ``kubectl rollout status`` command with the name of your cluster:
+
+   .. code:: bash
 
-     kubectl rollout status sts my-cluster-name-rs0
+      $ kubectl rollout status sts my-cluster-name-rs0
```

Comment:
lets mention here also how to perform the rotation of mongos and configsvrs automatically?